src/main/java/uk/ac/ic/wlgitbridge/bridge/repo/FSGitRepoStore.java

```diff
@@ -74,6 +75,32 @@ public ProjectRepo initRepo(String project) throws IOException {
                 ret, Optional.of(maxFileSize), Optional.empty());
     }
 
+     @Override
+     public ProjectRepo initRepoFromExisting(
+             String project, String fromProject
+     ) throws IOException {
+         String repoRoot = getRepoStorePath();
+         String sourcePath = repoRoot + "/" + fromProject;
+         String destinationPath = repoRoot + "/" + project;
+         Log.info("[{}] Init repo by copying data from: {}, to: {}",
+             project,
+             sourcePath,
+             destinationPath
+         );
+         try {
+             File source = new File(sourcePath);
+             File destination = new File(destinationPath);
+             FileUtils.copyDirectory(source, destination);
+         } catch (Exception e) {
```

Comment:
We could potentially not catch this exception here (i.e. remove the try-catch) and just let it bubble up, since `initRepoFromExisting` can already throw `IOException`. We'd still get the stack trace when the upstream method logs the failure, so I'm not sure what we gain by trapping it here. WDYT?

Reply:
Sure :+1:

Reply:
Ah, I think this bit here may be a vestigial leftover from development, I was catching an Exception from the copy operation, and then essentially re-throwing it as `IOException`, probably to avoid altering the Exceptions declared on every function up the stack.